src/components/CustomerQuota/NoQuotaCard.tsx

```diff
@@ -84,10 +95,29 @@ const NoPreviousTransactionTitle: FunctionComponent = () => (
   <AppText style={sharedStyles.statusTitle}>Limit reached.</AppText>
 );
 
+const AppealButton: FunctionComponent<AppealButton> = ({ onAppeal }) => {
+  return (
+    <TouchableOpacity
+      onPress={() => {
+        onAppeal();
+      }}
```

Comment:
Any issues with passing `onAppeal` directly to `onPress`?

Reply:
No issue. Will pass it directly